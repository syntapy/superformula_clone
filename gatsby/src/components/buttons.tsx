import * as React from "react"
import * as styles from "../styles/buttons.module.css"
import { StyleProps } from "../utils"

interface ButtonProps {
  href?: string
  text: string
}

interface DataCyProps {
  dataCy: string
}

export function A(props: ButtonProps & StyleProps) {
  const className: string = styles.a + " " + props.className
  return (
    <a className={className} href={props.href}>
      {props.text}
      {props.children}
    </a>
  )
}

export function HomeNavButton(props: ButtonProps & StyleProps & DataCyProps) {
  let className: string = styles.homeNavButton + " " + props.className
  return (
    <div data-cy={props.dataCy} className={className}>
      <A href="/">{props.text}</A>
    </div>
  )
}

export function NavButton(props: ButtonProps & StyleProps & DataCyProps) {
  const className: string = styles.navButton + " " + props.className
  return (
    <div data-cy={props.dataCy} className={className}>
      <A href={props.href}>{props.text}</A>
    </div>
  )
}

export function CareersNavButton(props: ButtonProps & StyleProps & DataCyProps) {
  const className: string = props.className + " " + styles.careersNavButton
  return (
    <div data-cy={props.dataCy} className={className}>
      <A className={className} href={props.href}>{props.text}</A>
    </div>
  )
}

interface HRefProps {
  href: string
}

function ButtonArrow(props: ButtonProps & StyleProps) {
  const className: string = styles.buttonArrow + " " + props.className
  return (
    <div className={className}>
      <A className={className} href={props.href}>{props.text} â†’ </A>
    </div>
  )
}

export function ButtonLight(props: ButtonProps) {
  const className: string = styles.buttonLight + " " + props.className
  return (
    <ButtonArrow className={className} href={props.href} text={props.text} />
  )
}

export function ButtonDark(props: ButtonProps) {
  const className: string = styles.buttonDark + " " + props.className
  return (
    <ButtonArrow className={className} href={props.href} text={props.text} />
  )
}

export function ArticleButton(props: ButtonProps) {
  const className: string = styles.articleButton + " " + props.className
  return (
    <div>
      <A className={className} href={props.href}>{props.text}</A>
    </div>
  )
}

interface SvgProps {
  icon: React.FunctionComponent<React.SVGProps<SVGSVGElement>>
  ariaLabel: string
}

export function SvgButton(props: StyleProps & SvgProps) {
  const className: string = styles.svgButton + " " + props.className
  const w: string = props.icon.props.width.toString() + "px"
  const h: string = props.icon.props.height.toString() + "px"
  const dimensionStyle: React.CSSProperties = {
    width: w,
    minWidth: w,
    height: h,
    minHeight: h
  }
  return (
    <button 
      style={dimensionStyle}
      className={className}
      aria-label={props.ariaLabel}
      onClick={props.onClick}
    >
      {props.icon}
    </button>
  )
}
