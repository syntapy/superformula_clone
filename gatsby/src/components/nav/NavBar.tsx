import * as React from "react"
import { useStaticQuery, graphql } from "gatsby"

import { HomeNavButton, SvgButton } from "../button/index.ts"

import type { NavItemInfo } from "./types"
import NavItems from "./NavItems.tsx"

// Menu Icons
import Chilidogmenu from "../../images/svg/chilidogmenu.svg"

import * as stylesMobile from "../../styles/navbar/mobile.module.css"
import * as stylesDesktop from "../../styles/navbar/desktop.module.css"
import * as styles from "../../styles/navbar/common.module.css"
import * as orientationStyles from "../../styles/utilities/orientation.module.css"
import { mobile } from "../../styles/utilities/visibility.module.css"

export default function NavBar() {
  const [mobileMenuActive, setMobileMenuActive] = React.useState(false)
  const [menuElement, setMenuElement] = React.useState(null)
  const data = useStaticQuery(
    graphql`
      query {
        contentfulLandingPage {
          title
        }
        allContentfulSection(
          sort: {sort_id: ASC}
          filter: {sort_id: {in: [5, 9, 10, 11, 12, 13]}}
        ) {
          nodes {
            linkHref
            linkText
            title
          }
        }
      }
    `)

  const title: string = data.contentfulLandingPage.title
  const servicesHref: string = data.allContentfulSection.nodes[0].linkHref
  const servicesText: string = data.allContentfulSection.nodes[0].title
  const workHref: string = data.allContentfulSection.nodes[1].linkHref
  const workText: string = data.allContentfulSection.nodes[1].linkText
  const articlesHref: string = data.allContentfulSection.nodes[2].linkHref
  const articlesText: string = data.allContentfulSection.nodes[2].linkText
  const contactHref: string = data.allContentfulSection.nodes[3].linkHref
  const contactText: string = data.allContentfulSection.nodes[3].linkText
  const careersHref: string = data.allContentfulSection.nodes[4].linkHref
  const careersText: string = data.allContentfulSection.nodes[4].linkText
  const aboutHref: string = data.allContentfulSection.nodes[5].linkHref
  const aboutText: string = data.allContentfulSection.nodes[5].linkText
  const navMobile: string = ""

  const items: NavItemInfo[] = []

  items.push({ href: workHref, text: workText })
  items.push({ href: servicesHref, text: servicesText })
  items.push({ href: articlesHref, text: articlesText })
  items.push({ href: contactHref, text: contactText })
  items.push({ href: careersHref, text: careersText })
  items.push({ href: aboutHref, text: aboutText })

  // Styles
  const navStyle: string = styles.navbar
      + " " + stylesMobile.navbar

  const navItemsStyle: string = stylesDesktop.navItems
      + " " + stylesMobile.navItems
      + " " + stylesMobile.navItemsHidden
      + " " + orientationStyles.mobileV_desktopH
  const chilidogStyle: string = stylesMobile.chilidog + " " + mobile
  const menuId: string = "nav-menu"

  function onChilidogClick(): void {
    // omfg !!!
    try {
      document
    } catch(error) {
      console.error("Document undefined in NavBar component's `onChilidogClick` function")

      return
    }

    if (!mobileMenuActive) {
      const menu: HTMLElement | null = document.getElementById(menuId)
      if (menu || !!menu) {
        menu.classList.remove(stylesMobile.navItemsHidden)
        menu.classList.add(stylesMobile.navItemsActive)
        setMobileMenuActive(true)
      }
    }
  }

  return (
    <nav className={navStyle}>
      <HomeNavButton text={title}/>
      <NavItems
        className={navItemsStyle}
        menuId={menuId}
        title={title}
        items={items}
        mobileMenuActive={mobileMenuActive}
        setMobileMenuActive={setMobileMenuActive}
      />
      <SvgButton
        className={chilidogStyle}
        onClick={onChilidogClick}
        aria-label="Open Navbar"
        title="Open Navbar"
        icon={<Chilidogmenu width={32} height={45} data-cy="chilidog-svg" />}
      />
    </nav>
  )
}
