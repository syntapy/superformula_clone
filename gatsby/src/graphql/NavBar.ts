import { useStaticQuery, graphql } from "gatsby"

export default getData() {
  const data = useStaticQuery(
    graphql`
      query {
        contentfulLandingPage {
          title
        }
        allContentfulSection(
          sort: {sort_id: ASC}
          filter: {sort_id: {in: [5, 9, 10, 11, 12, 13]}}
        ) {
          nodes {
            linkHref
            linkText
            title
          }
        }
      }
    `)

  const title: string = data.contentfulLandingPage.title
  const servicesHref: string = data.allContentfulSection.nodes[0].linkHref
  const servicesText: string = data.allContentfulSection.nodes[0].title
  const workHref: string = data.allContentfulSection.nodes[1].linkHref
  const workText: string = data.allContentfulSection.nodes[1].linkText
  const articlesHref: string = data.allContentfulSection.nodes[2].linkHref
  const articlesText: string = data.allContentfulSection.nodes[2].linkText
  const contactHref: string = data.allContentfulSection.nodes[3].linkHref
  const contactText: string = data.allContentfulSection.nodes[3].linkText
  const careersHref: string = data.allContentfulSection.nodes[4].linkHref
  const careersText: string = data.allContentfulSection.nodes[4].linkText
  const aboutHref: string = data.allContentfulSection.nodes[5].linkHref
  const aboutText: string = data.allContentfulSection.nodes[5].linkText
  const navMobile: string = ""

  const items: NavItemInfo[] = []

  items.push({ href: workHref, text: workText })
  items.push({ href: servicesHref, text: servicesText })
  items.push({ href: articlesHref, text: articlesText })
  items.push({ href: contactHref, text: contactText })
  items.push({ href: careersHref, text: careersText })
  items.push({ href: aboutHref, text: aboutText })

  return items
}
