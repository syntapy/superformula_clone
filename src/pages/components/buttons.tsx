import * as React from "react"
import * as styles from "../styles/buttons.module.css"

interface StyleProps {
  className?: string
}

interface ButtonProps {
  href: string
  text: string
}

export function A(props: ButtonProps & StyleProps) {
  const className: string = styles.a + " " + props.className
  return (
    <a className={className} href={props.href}>
      {props.text}
      {props.children}
    </a>
  )
}

export function HomeNavButton(props: ButtonProps & StyleProps) {
  let className: string = styles.homeNavButton + " " + props.className
  return (
    <div className={className}>
      <A className={className} href={props.href}>{props.text}</A>
    </div>
  )
}

export function NavButton(props: ButtonProps & StyleProps) {
  const className: string = styles.navButton + " " + props.className
  console.log(className)
  return (
    <div className={className}>
      <A className={className} href={props.href}>{props.text}</A>
    </div>
  )
}

export function CareersNavButton(props: ButtonProps & StyleProps) {
  const className: string = styles.careersNavButton + " " + props.className
  return (
    <div className={className}>
      <A className={className} href={props.href}>{props.text}</A>
    </div>
  )
}

interface HRefProps {
  href: string
}

function ButtonArrow(props: ButtonProps & StyleProps) {
  const className: string = styles.buttonArrow + " " + props.className
  return (
    <div className={className}>
      <A className={className} href={props.href}>{props.text} â†’ </A>
    </div>
  )
}

export function ButtonLight(props: ButtonProps) {
  const className: string = styles.buttonLight + " " + props.className
  return (
    <ButtonArrow className={className} href={props.href} text={props.text} />
  )
}

export function ButtonDark(props: ButtonProps) {
  const className: string = styles.buttonDark + " " + props.className
  return (
    <ButtonArrow className={className} href={props.href} text={props.text} />
  )
}

export function ArticleButton(props: ButtonProps) {
  const className: string = styles.articleButton + " " + props.className
  return (
    <div>
      <A className={className} href={props.href}>{props.text}</A>
    </div>
  )
}

interface SvgProps {
  icon: string
  alt: string
}

export function ButtonSvg(props: SvgProps & HRefProps) {
  return (
    <button href={props.href}>
    </button>
  )
}
