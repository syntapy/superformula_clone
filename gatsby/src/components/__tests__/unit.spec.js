import React from "react"
import renderer from "react-test-renderer"

import Header from "../header"
import { HomeNavButton } from "../buttons"
import NavBar from "../navbar"

const isMobileTrue = jest.fn(() => true)

describe("Header", () => {
	it("renders correctly", () => {
		const tree = renderer
			.create(<Header siteTitle="Default Starter" isMobile={isMobileTrue} />)
			.toJSON()
		expect(tree).toMatchSnapshot()
	})
})

describe("HomeNavButton", () => {
	it("renders correctly", () => {
		const tree = renderer
			.create(<HomeNavButton href={"test"} text={"test url"}>Test Url</HomeNavButton>)
			.toJSON()
		expect(tree).toMatchSnapshot()
	})
})

describe("NavBar", () => {
	it("renders correctly", () => {
		const tree = renderer
			.create(<NavBar isMobile={isMobileTrue} />)
			.toJSON()
		expect(tree).toMatchSnapshot()
	})
})
