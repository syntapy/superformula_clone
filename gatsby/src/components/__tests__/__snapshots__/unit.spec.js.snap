// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header>
  <div
    className="src-styles-fixed-CDvAcKBA-T src-styles-navbarContainer-DN8BHCiz-C"
    id="desktop-nav-wrapper"
  >
    <div
      className="src-styles-container-8sTzKuoBNk src-styles-navbar-m-v7wwUaSW"
    >
      <button
        className="src-styles-svgButton-P5syo6dygz src-styles-chilidog-pBCE7RY2-t"
        onClick={[Function]}
        style={
          {
            "height": "45px",
            "minHeight": "45px",
            "minWidth": "32px",
            "width": "32px",
          }
        }
      >
        <svg
          fill="#ffffff"
          height="45"
          viewBox="0 0 80 50"
          width="32"
        >
          <rect
            height="5"
            width="44"
          />
          <rect
            height="5"
            width="80"
            y="20"
          />
          <rect
            height="5"
            width="50"
            y="40"
          />
        </svg>
      </button>
      <div
        className="src-styles-homeNavButton-FOVN2eOxu- src-styles-homeItem-bdSEHrdf-I"
      >
        <a
          className="src-styles-a-g07BlHX8bK src-styles-homeNavButton-FOVN2eOxu- src-styles-homeItem-bdSEHrdf-I"
          href="/"
        >
          Superformula.
        </a>
      </div>
      <nav
        className="src-styles-navItem-MrimwUo-Wg src-styles-pageItemList-zsb5979aQc"
        id="desktop-nav"
      >
        <div
          className="src-styles-navbarContainer-DN8BHCiz-C src-styles-itemListHomeItem-9NkznCL7LS"
        >
          <button
            className="src-styles-svgButton-P5syo6dygz src-styles-menuClose-J96rIZBjBX"
            style={
              {
                "height": "40px",
                "minHeight": "40px",
                "minWidth": "40px",
                "width": "40px",
              }
            }
          >
            <svg
              fill="#ffffff"
              height="40"
              viewBox="0 0 80 50"
              width="40"
            >
              <rect
                height="5"
                width="44"
              />
              <rect
                height="5"
                width="80"
                y="20"
              />
              <rect
                height="5"
                width="50"
                y="40"
              />
            </svg>
          </button>
          <div
            className="src-styles-homeNavButton-FOVN2eOxu- src-styles-home-lovhuGmMZH"
          >
            <a
              className="src-styles-a-g07BlHX8bK src-styles-homeNavButton-FOVN2eOxu- src-styles-home-lovhuGmMZH"
              href="/"
            >
              Superformula.
            </a>
          </div>
        </div>
        <div
          className="src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
        >
          <a
            className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
            href="/services"
          >
            Services
          </a>
        </div>
        <div
          className="src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
        >
          <a
            className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
            href="/work"
          >
            Work
          </a>
        </div>
        <div
          className="src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
        >
          <a
            className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
            href="/articles"
          >
            Articles
          </a>
        </div>
        <div
          className="src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
        >
          <a
            className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
            href="/contact"
          >
            Contact
          </a>
        </div>
        <div
          className="src-styles-careersNavButton-sC5v2hJNN4 src-styles-navItem-MrimwUo-Wg"
        >
          <a
            className="src-styles-a-g07BlHX8bK src-styles-careersNavButton-sC5v2hJNN4 src-styles-navItem-MrimwUo-Wg"
            href="https://careers.superformula.com"
          >
            Careers
          </a>
        </div>
      </nav>
    </div>
  </div>
  <nav
    className="src-styles-navItem-MrimwUo-Wg src-styles-pageItemListMobile-OWSYDLp7Qb"
    id="mobile-nav"
  >
    <div
      className="src-styles-navbarContainer-DN8BHCiz-C src-styles-itemListHomeItem-9NkznCL7LS"
    >
      <button
        className="src-styles-svgButton-P5syo6dygz src-styles-menuClose-J96rIZBjBX"
        onClick={[Function]}
        style={
          {
            "height": "40px",
            "minHeight": "40px",
            "minWidth": "40px",
            "width": "40px",
          }
        }
      >
        <svg
          fill="#ffffff"
          height="40"
          viewBox="0 0 80 50"
          width="40"
        >
          <rect
            height="5"
            width="44"
          />
          <rect
            height="5"
            width="80"
            y="20"
          />
          <rect
            height="5"
            width="50"
            y="40"
          />
        </svg>
      </button>
      <div
        className="src-styles-homeNavButton-FOVN2eOxu- src-styles-home-lovhuGmMZH"
      >
        <a
          className="src-styles-a-g07BlHX8bK src-styles-homeNavButton-FOVN2eOxu- src-styles-home-lovhuGmMZH"
          href="/"
        >
          Superformula.
        </a>
      </div>
    </div>
    <div
      className="src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
    >
      <a
        className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
        href="/services"
      >
        Services
      </a>
    </div>
    <div
      className="src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
    >
      <a
        className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
        href="/work"
      >
        Work
      </a>
    </div>
    <div
      className="src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
    >
      <a
        className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
        href="/articles"
      >
        Articles
      </a>
    </div>
    <div
      className="src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
    >
      <a
        className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
        href="/contact"
      >
        Contact
      </a>
    </div>
    <div
      className="src-styles-careersNavButton-sC5v2hJNN4 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
    >
      <a
        className="src-styles-a-g07BlHX8bK src-styles-careersNavButton-sC5v2hJNN4 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
        href="https://careers.superformula.com"
      >
        Careers
      </a>
    </div>
  </nav>
</header>
`;

exports[`HomeNavButton renders correctly 1`] = `
<div
  className="src-styles-homeNavButton-FOVN2eOxu- undefined"
>
  <a
    className="src-styles-a-g07BlHX8bK src-styles-homeNavButton-FOVN2eOxu- undefined"
    href="test"
  >
    test url
  </a>
</div>
`;

exports[`NavBar renders correctly 1`] = `
[
  <div
    className="src-styles-fixed-CDvAcKBA-T src-styles-navbarContainer-DN8BHCiz-C"
    id="desktop-nav-wrapper"
  >
    <div
      className="src-styles-container-8sTzKuoBNk src-styles-navbar-m-v7wwUaSW"
    >
      <button
        className="src-styles-svgButton-P5syo6dygz src-styles-chilidog-pBCE7RY2-t"
        onClick={[Function]}
        style={
          {
            "height": "45px",
            "minHeight": "45px",
            "minWidth": "32px",
            "width": "32px",
          }
        }
      >
        <svg
          fill="#ffffff"
          height="45"
          viewBox="0 0 80 50"
          width="32"
        >
          <rect
            height="5"
            width="44"
          />
          <rect
            height="5"
            width="80"
            y="20"
          />
          <rect
            height="5"
            width="50"
            y="40"
          />
        </svg>
      </button>
      <div
        className="src-styles-homeNavButton-FOVN2eOxu- src-styles-homeItem-bdSEHrdf-I"
      >
        <a
          className="src-styles-a-g07BlHX8bK src-styles-homeNavButton-FOVN2eOxu- src-styles-homeItem-bdSEHrdf-I"
          href="/"
        >
          Superformula.
        </a>
      </div>
      <nav
        className="src-styles-navItem-MrimwUo-Wg src-styles-pageItemList-zsb5979aQc"
        id="desktop-nav"
      >
        <div
          className="src-styles-navbarContainer-DN8BHCiz-C src-styles-itemListHomeItem-9NkznCL7LS"
        >
          <button
            className="src-styles-svgButton-P5syo6dygz src-styles-menuClose-J96rIZBjBX"
            style={
              {
                "height": "40px",
                "minHeight": "40px",
                "minWidth": "40px",
                "width": "40px",
              }
            }
          >
            <svg
              fill="#ffffff"
              height="40"
              viewBox="0 0 80 50"
              width="40"
            >
              <rect
                height="5"
                width="44"
              />
              <rect
                height="5"
                width="80"
                y="20"
              />
              <rect
                height="5"
                width="50"
                y="40"
              />
            </svg>
          </button>
          <div
            className="src-styles-homeNavButton-FOVN2eOxu- src-styles-home-lovhuGmMZH"
          >
            <a
              className="src-styles-a-g07BlHX8bK src-styles-homeNavButton-FOVN2eOxu- src-styles-home-lovhuGmMZH"
              href="/"
            >
              Superformula.
            </a>
          </div>
        </div>
        <div
          className="src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
        >
          <a
            className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
            href="/services"
          >
            Services
          </a>
        </div>
        <div
          className="src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
        >
          <a
            className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
            href="/work"
          >
            Work
          </a>
        </div>
        <div
          className="src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
        >
          <a
            className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
            href="/articles"
          >
            Articles
          </a>
        </div>
        <div
          className="src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
        >
          <a
            className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-navItem-MrimwUo-Wg"
            href="/contact"
          >
            Contact
          </a>
        </div>
        <div
          className="src-styles-careersNavButton-sC5v2hJNN4 src-styles-navItem-MrimwUo-Wg"
        >
          <a
            className="src-styles-a-g07BlHX8bK src-styles-careersNavButton-sC5v2hJNN4 src-styles-navItem-MrimwUo-Wg"
            href="https://careers.superformula.com"
          >
            Careers
          </a>
        </div>
      </nav>
    </div>
  </div>,
  <nav
    className="src-styles-navItem-MrimwUo-Wg src-styles-pageItemListMobile-OWSYDLp7Qb"
    id="mobile-nav"
  >
    <div
      className="src-styles-navbarContainer-DN8BHCiz-C src-styles-itemListHomeItem-9NkznCL7LS"
    >
      <button
        className="src-styles-svgButton-P5syo6dygz src-styles-menuClose-J96rIZBjBX"
        onClick={[Function]}
        style={
          {
            "height": "40px",
            "minHeight": "40px",
            "minWidth": "40px",
            "width": "40px",
          }
        }
      >
        <svg
          fill="#ffffff"
          height="40"
          viewBox="0 0 80 50"
          width="40"
        >
          <rect
            height="5"
            width="44"
          />
          <rect
            height="5"
            width="80"
            y="20"
          />
          <rect
            height="5"
            width="50"
            y="40"
          />
        </svg>
      </button>
      <div
        className="src-styles-homeNavButton-FOVN2eOxu- src-styles-home-lovhuGmMZH"
      >
        <a
          className="src-styles-a-g07BlHX8bK src-styles-homeNavButton-FOVN2eOxu- src-styles-home-lovhuGmMZH"
          href="/"
        >
          Superformula.
        </a>
      </div>
    </div>
    <div
      className="src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
    >
      <a
        className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
        href="/services"
      >
        Services
      </a>
    </div>
    <div
      className="src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
    >
      <a
        className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
        href="/work"
      >
        Work
      </a>
    </div>
    <div
      className="src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
    >
      <a
        className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
        href="/articles"
      >
        Articles
      </a>
    </div>
    <div
      className="src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
    >
      <a
        className="src-styles-a-g07BlHX8bK src-styles-navButton-7LzWMSAco3 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
        href="/contact"
      >
        Contact
      </a>
    </div>
    <div
      className="src-styles-careersNavButton-sC5v2hJNN4 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
    >
      <a
        className="src-styles-a-g07BlHX8bK src-styles-careersNavButton-sC5v2hJNN4 src-styles-pageItem-LqEbulJmM- src-styles-navItem-MrimwUo-Wg"
        href="https://careers.superformula.com"
      >
        Careers
      </a>
    </div>
  </nav>,
]
`;
